server.js

```diff
@@ -1,4 +1,6 @@
 const port = process.env.PORT || 3000;
+const polling_interval = (process.env.POLLING_INTERVAL && parseInt(process.env.POLLING_INTERVAL)) || 2000;
```

Comment:
Camel me plz

Reply:
Outra coisa é que realmente prefiro passar a base para o `parseInt`

`parseInt(xxx, 10)`

Talvez o node não faça a inferência como antigos browsers, mas oficialmente não acho nenhuma doc dizendo sobre isso, o melhor que temos é o mdn * For this reason always specify a radix when using parseInt*

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt